pkg/virt-operator/install-strategy/strategy.go

```diff
@@ -273,28 +273,42 @@ func GenerateCurrentInstallStrategy(config *operatorutil.KubeVirtDeploymentConfi
 		}
 	}
 
+	var productName string
+	var productVersion string
+
+	if isValidLabel(config.AdditionalProperties[operatorutil.ProductNameKey]) {
```

Comment:
It doesn't matters since `AdditionalProperties` and all the other fields are public/exported for config, but it seems the convention that is being used is to add helper methods to access each of it's fields. Maybe we should do the same for `GetProductName()` and `GetProductVersion()`. It does have the additional benefit of making them more discoverable since they are hidden under the additionalproperties map

Reply:
+1 thanks for the insight!